modules/installation-azure-trusted-launch.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-network-customizations
+
+:_content-type: PROCEDURE
+[id="installation-azure-trusted-launch_{context}"]
+= Enabling trusted launch for Azure VMs
```

Comment:
It's decided to be a TP in 4.14, do we need to highlight this in doc, or is it enough to mention only in RN?

Reply:
Added Tech preview snippet